ingestion/tests/integration/ometa/test_ometa_service_api.py

```diff
@@ -55,6 +56,16 @@ class OMetaServiceTest(TestCase):
     )
     metadata = OpenMetadata(server_config)
 
+    # use ingestion_bot jwt token for this test
```

Comment:
we might want to add here that we need to use the bot because admin won't see the pwd?